view/admin.phtml

```diff
@@ -323,6 +323,32 @@
                         </p>
                     </td>
                 </tr>
+                <tr>
+                    <th>
+                        <label for="authLDAPExtUsers">EXPERIMENTAL Enable LDAP Group Lookup for 'External' users?</label>
+                    </th>
+                    <td>
+                        <input type="checkbox" name="authLDAPExternalUsers" id="authLDAPExternalUsers" value="1"<?php echo $tExtChecked; ?>/>
+                        <p class="description">
+                            Shall we try to find the user in LDAP even if they logged in through a different mechanism (SAML, OIDC, social logins, local accounts, etc)? CAUTION: This has the potential to break access for users,
+			                depending on the configuration of other settings.
+                        </p>
+                        <p class="description">This should only be checked if you know what you are doing! THIS WILL TRIGGER EVEN IF LDAP AUTHENTICATION IS DISABLED.</p>
```

Comment:
The "Enable AuthLDAP" should enable the whole plugin! Not only the Authentication. It is a safety catch to disable all LDAP-interaction in case that becomes necessaary. 

To disable the authentication part of authLdap we need a separate flag to "Disable Authentication via LDAP"

Reply:
I'm 100% fine with this. Wasn't sure how you wanted to approach things, so I didn't add that in, but you're absolutely right - that's definitely needed as a safety switch.